routers/web/repo/issue_timetrack.go

```diff
@@ -42,7 +42,7 @@ func AddTimeManually(c *context.Context) {
 		return
 	}
 
-	if _, err := issues_model.AddTime(c.Doer, issue, int64(total.Seconds()), time.Now()); err != nil {
+	if _, err := issues_model.AddTime(c, c.Doer, issue, int64(total.Seconds()), time.Now()); err != nil {
```

Comment:
`c`?
Looks quite weird compared to the remaining `ctx`…

Reply:
Yes, but I keep it not changed because it's not this PR's responsibility.